doc/build-openbsd.md

```diff
@@ -21,23 +18,23 @@ pkg_add boost
 git clone https://github.com/bitcoin/bitcoin.git
 ```
 
+Optional: To build with Qt 5 support, you also need qt5 and protobuf:
+```bash
+pkg_add protobuf
+pkg_add qt5
+```
+
 See [dependencies.md](dependencies.md) for a complete overview.
 
-**Important**: From OpenBSD 6.2 onwards a C++11-supporting clang compiler is
```

Comment:
Any specific reason to remove this warning?

Reply:
To keep the guide as short an concise as possible. I don't see any reason why to mention it?

Reply:
Because people have run into this issue by forgetting to add those CC= suffixes for part of the build, so it is useful to mention it here.